packages/node_modules/nav-frontend-ekspanderbartpanel/src/ekspanderbartpanel-base.tsx

```diff
@@ -4,12 +4,11 @@ import { UnmountClosed, Collapse, CollapseProps } from 'react-collapse';
 import 'nav-frontend-ekspanderbartpanel-style';
 import { guid, keyCodes } from 'nav-frontend-js-utils';
 
-const cls = (props) =>
-    classnames('ekspanderbartPanel', props.className, {
-        'ekspanderbartPanel--lukket': !props.apen,
-        'ekspanderbartPanel--apen': props.apen,
-        'ekspanderbartPanel--border': props.border
-    });
+const cls = (props) => classnames('ekspanderbartPanel', props.className, {
+    'ekspanderbartPanel--lukket': !props.apen,
+    'ekspanderbartPanel--apen': props.apen,
+    'ekspanderbartPanel--border': props.border,
```

Comment:
Endret på reglen om dangle-comma? Bør isåfall også gjøres for javascript

Reply:
Ja den var vist endret for Ts, men var skrevet feil så ble ikke kjørt og oppdaget. Fikset for Js og Ts nå